system-tests/src/main/java/io/apicurio/registry/systemtests/registryinfra/resources/ApicurioRegistryResourceType.java

```diff
@@ -16,7 +17,7 @@
 public class ApicurioRegistryResourceType implements ResourceType<ApicurioRegistry> {
     @Override
     public Duration getTimeout() {
-        return Duration.ofMinutes(3);
+        return Duration.ofMinutes(10);
```

Comment:
This looks really excessive, can we understand what is taking so long, or, in the worst case, add a comment on why this needs to be this long.